hack/generate-deploy-scripts.sh

```diff
@@ -163,6 +163,8 @@ controller:
       service.beta.kubernetes.io/do-loadbalancer-enable-proxy-protocol: "true"
   config:
     use-proxy-protocol: "true"
+  admissionWebhooks:
+    timeoutSeconds: 29
```

Comment:
Does 29 have any special meaning? Will there be any abnormalities if I use 25 or 30?  @boukeversteegh

Reply:
That being said, I suppose 29s is a very high timeout for a web-request, but I don't understand the consequences of changing this value.

Reply:
No other adverse effects were found, but it was indeed useful.  Leave my LGTM

Reply:
Awesome! Thank you!